cx2t/src/claw/ClawX2T.java

```diff
@@ -274,18 +283,26 @@ public static void main(String[] args) throws Exception {
       Configuration.get().setForcePure();
     }
 
+    ClawTranslatorDriver translatorDriver;
+
     // Call the translator driver to apply transformation on XcodeML/F
-    ClawTranslatorDriver translatorDriver =
-        new ClawTranslatorDriver(input, xcmlOutput);
+    if(recipeScript != null){
+      // Transformation is to be performed by Python script
+      translatorDriver = 
+          new ClawPythonTranslatorDriver(recipeScript, input, xcmlOutput);
+
+    } else {
+      translatorDriver = new ClawTranslatorDriver(input, xcmlOutput);
+    }
     translatorDriver.analyze();
     translatorDriver.transform();
     translatorDriver.flush();
 
-    // Produce report
-    if(cmd.hasOption("r")) {
+    // Produce report (unless we've used the Python driver)
+    if(recipeScript == null && cmd.hasOption("r")) {
       ClawTransformationReport report =
           new ClawTransformationReport(cmd.getOptionValue("r"));
-      report.generate(args, translatorDriver);
+      report.generate(args, (ClawTranslatorDriver)translatorDriver);
```

Comment:
Since `translatorDriver` is already a `ClawTranslatorDriver`, I don't think the case is necessary there.

Reply:
Oh, good point. Removed the cast.